packages/github/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/github",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
We should update `Releases.MD` for all of these packages as well

Reply:
Yep.  Per IM, I said I was going to finish this PR tomorrow.  I converted to a draft to make that clearer.